packages/pages/src/common/src/project/structure.ts

```diff
@@ -17,6 +18,12 @@ export interface ProjectFilepaths {
   hydrationBundleOutputRoot: string;
   /** The folder path where the compiled server bundles should go */
   serverBundleOutputRoot: string;
+  /**
+   * The domain of the site, which also serve as the subfolder name
+   * inside {@link templatesRoot} and {@link sitesConfigRoot}.
+   * This is used for the case of multibrand setup within a single repo.
```

Comment:
I would put this sentence first. By having it first, people can just skip the rest of the description if they're not doing multi-brand.